vignettes/datasets.Rmd

```diff
@@ -26,8 +26,10 @@ To create new datasets, multiple paths exist. In the web application, you can up
 
 You can create a dataset from any `data.frame` you have in your R session with `newDataset`:
 
-```{r, eval=FALSE}
-load("economist.RData")
+This package provides one as `economist` which loads a `data.frame` named `df` because old habits die hard.
+
+```{r}
+data("economist", package = "crunch") # named df
```

Comment:
Almost. Also need to edit the line that follows this (here, in datasets.Rmd), which is `dim(df)`, as well as L44 of https://github.com/Crunch-io/rcrunch/blob/master/vignettes/variables.Rmd.